src/core/locator/qgslocatorfilter.h

```diff
@@ -207,6 +207,13 @@ class CORE_EXPORT QgsLocatorFilter : public QObject
      */
     virtual void triggerResult( const QgsLocatorResult &result ) = 0;
 
+    /**
+     * This method will be called on main thread on the original filter (not a clone).
+     * This should be reimplemented to clear any change made by a former call to triggerResult.
+     * \since QGIS 3.2
```

Comment:
I like it! :+1: 

I'd just add an example here - "For instance, this can be used to remove any on-canvas rubber bands which have been created when a previous search result was triggered"